### ISSUE TITLE
Bring nostrify to npm also

### ISSUE THREAD

**Issue by antonioconselheiro** [soapbox-pub/nostrify#6]
I would like to suggest to include @nostrify/nostrifyin `npm` repository.
Always I try to update @nostrify/nostrify using `npm install @nostrify/nostrify@latest --save` it brokes.
When I try to auto update angular and it dependencies by using `ng update` it brokes when trying to update @nostrify/nostrify, so I must remove the package always I need to update the frameworks libraries and then reinstall.

File .npmrc is included and configured, but these errors keep happening.

![image](https://github.com/user-attachments/assets/38749bac-9c0d-48ce-9aef-513157122d25)

**Comment by alexgleason** [comment]
The problem with npm is that I need to add a build step. With jsr I can push my TypeScript code directly to jsr and it "just works". I'm using jsr specifically because I don't want that extra complexity.

I would encourage you to give Deno a try. It's made by the creator of Node, and as of the recent 2.0 release I would say it's finally "better" than Node: https://deno.com/blog/v2.0 I think Deno and jsr are the future of JavaScript, so I am wanting to move forwards not backwards.

**Comment by antonioconselheiro** [comment]
You look decided into this.
I believe a in retrocompatible way  to move, `ng update` of angular does not make a good work in auto update projects with cross repository, or maybe I have missed something.

I believe most people and enterprises will migrate from node/npm slowly to jsr/deno, to keep my retrocompatible way  to move, I ask you: is no problem for you if I fork this project and publish in npm with my custom library contexts (@belomonte) so I can point this as my dependency and not something in jsr?

**Comment by alexgleason** [comment]
> I ask you: is no problem for you if I fork this project and publish in npm with my custom library contexts

Sure, that's no problem.

> I believe a in retrocompatible way to move, `ng update` of angular does not make a good work in auto update projects with cross repository, or maybe I have missed something.

I was thinking we should create an issue in Angular or Deno to see if they can solve the problem.

But then I looked at [`ng update`](https://angular.dev/cli/update) and I'm simply trying to understand it... if I understand correctly, this is not a package manager? However, it uses a package manager? So it must detect whether you are using `npm` or `yarn` etc automatically. Then it needs to support an infinite number of package managers like `pnpm`, `bun`, etc. This is cursed.

It seems like somebody caused themselves pain writing this script instead of letting the package manager do it. And by depending on it you are "locked in" to it. Free your chains. :joy:

**Comment by antonioconselheiro** [comment]
You're right. I'll see if there is no issue in angular about it and open it, I believe they are using npm to install the packages but for some reason they don't read .npmrc configs.

About ng update:
It update the new angular packages, update libraries and change your code to keep it compatible. It also recommend migrations to new angular content. Angular have a update checklist to help update things that `ng update` can't do: https://update.angular.dev/

Thank you for your time.
